docs/guides/1238-project-file-organization.md

```diff
@@ -0,0 +1,242 @@
+---
+slug: /1238/project-file-organization
+displayed_sidebar: "0.2"
+---
+
+# Project file organization
+
+As time will pass, your actions will get bigger and bigger. You will feel the need to better organize your file structure by moving some parts in subfolders.
+
+The easiest way, when iterating fast, is to rely on the root module.
```

Comment:
`the simplest next step is to rely on the root module`

What is the root module? I think that we need to answer the root module here.